Source/BoundaryConditions/PML.H

```diff
@@ -104,6 +104,7 @@ public:
          int ncell, int delta, amrex::IntVect ref_ratio,
          amrex::Real dt, int nox_fft, int noy_fft, int noz_fft, bool do_nodal,
          int do_moving_window, int pml_has_particles, int do_pml_in_domain,
+         const bool psatd_linear_in_J,
```

Comment:
Discussed in our meeting:
maybe better `psatd_linear_in_time`

Also: let us add doxygen strings that explain these flags/options :)
e.g. this one will use two J fields